include/verilatedos.h

```diff
@@ -430,6 +442,11 @@ using ssize_t = uint32_t;  ///< signed size_t; returned from read()
     Type(const Type& other) = delete; \
     Type& operator=(const Type&) = delete
 
+// Declare a class as unmovable; put after a private:
+#define VL_UNMOVABLE(Type) \
```

Comment:
Should VL_UNCOPYABLE also imply this?  Or vice-versa?  How many places need not both?

Reply:
I think it shouldn't. When object is ``VL_UNCOPYABLE`` usually it is fine to move it. I briefly searched code and there is only one class that deletes both copy and move constructor (and it is being added in this PR).